### ISSUE TITLE
why my loss  decrease under 0

### ISSUE THREAD

**Issue by tomsonsgs** [chilynn/sequence-labeling#25]
loss is supposed to be close to and bigger than  0, self.loss = - (self.target_path_score - self.total_path_score),target_path_score should be smaller than total_path_score

**Comment by tomsonsgs** [comment]
i solve the problem by change the code if condition "i==0" by "j==0" in getTransition(y_train_batch) function,and loss come to 0 as expected,you all can check on that，and will this improve the final accuracy?someone can try that

**Comment by tomsonsgs** [comment]
@chilynn you can check on that and see if it is correct
